examples/version-migration/same-container/tests/sameContainer.test.ts

```diff
@@ -42,6 +41,7 @@ describe("inventoryList", () => {
 			await expect(containsOne).toEqual(true);
 		});
 
+		// Test skipped, not functioning currently
```

Comment:
These were copied from separate-container migration test but the feature did not progress to a point where they would pass.  If/when we pick this back up we should still make them pass.